### ISSUE TITLE
Please add more examples

### ISSUE THREAD

**Issue by Bluscream** [quamotion/madb#164]
the docs are a bit sparse. also link to http://madb.bit13.com/ please so one can at least see the available stuff even if it does not tell how to use it

**Comment by Bluscream** [comment]
@qmfrederik could you show here how subscribing to the battery event would work?

**Comment by WimVriend** [comment]
I totally agree: the number of useful examples (all over the internet) is ridiculously low. To be exact: I only found the 2 examples on the madb project-page useful.

Some examples of, for instance, forwarding a TCP port and sending/receiving data would be welcome...
